auguste/roster.json

```diff
@@ -225,11 +225,15 @@
       ]
     },
     {
-      "name": "Nadir Khan",
+      "name": "Nadir Ahmed Khan",
       "github": "Nadirkhan98",
       "materials": [
         {
           "type": "cv"
+        },
+        {
+          "type": "portfolio",
+          "url": "https://app.netlify.com/sites/nadirkhan98-port/overview"
```

Comment:
The url shows

<img width="554" alt="image" src="https://user-images.githubusercontent.com/48137278/175490571-8ecabcbb-4d2e-4252-a4cd-b6b926c872c3.png">


Please replace with a url/website that is live